app/models/user.rb

```diff
@@ -1,5 +1,6 @@
 class User < ActiveRecord::Base
   authenticates_with_sorcery!
+  before_create :create_organiser_token
```

Comment:
Saving role as 1, even before organiser fills the form, the new organiser is displayed in list of added organisers. So is it required or should i leave ? Also how to diffrentiate added and pending organiser ? Next task is asana is  to display list of pending organisers.